selfdrive/ui/ui.h

```diff
@@ -117,7 +117,9 @@ typedef struct UIScene {
   uint64_t started_frame;
 } UIScene;
 
+class QUIState;
```

Comment:
Can we just remove the `typedef struct`? That's a leftover from the original c days.